grafana-plugin/src/containers/UserSettings/parts/tabs/PhoneVerification/PhoneVerification.tsx

```diff
@@ -168,7 +169,9 @@ export const PhoneVerification = observer((props: PhoneVerificationProps) => {
   const isButtonDisabled =
     phone === user.verified_phone_number ||
     (!isCodeSent && !isPhoneValid && !isPhoneCallInitiated) ||
-    !isPhoneProviderConfigured;
+    !isPhoneProviderConfigured ||
+    !window.grecaptcha;
+  const disabledButtonTooltipText = !window.grecaptcha ? 'reCAPTCHA has not been loaded' : undefined;
```

Comment:
Up to you but it's def easier reading it with `window.greptcha ? undefined : '...'`